pkg/hints/hint_utils/bigint_utils.go

```diff
@@ -3,12 +3,51 @@ package hint_utils
 import (
 	"math/big"
 
+	"github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
 	. "github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
+	. "github.com/lambdaclass/cairo-vm.go/pkg/types"
 	. "github.com/lambdaclass/cairo-vm.go/pkg/vm"
+	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
 	. "github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
 	"github.com/pkg/errors"
 )
 
```

Comment:
This file (aka pkg/hints/hint_utils/bigint_utils.go) is used exclusively to support big integer struct representations used across various hints, it shouldn't contain hint implementations. Please move hint implementations to the appropiate file under the 'pkg/hints' directory